### ISSUE TITLE
API regression in dev and beta - options are *required*

### ISSUE THREAD

**Issue by lloyd** [mozilla/persona#1442]
STR:
1. invoke navigator.id.get(function() { });

notice error:

```
Uncaught TypeError: Cannot set property 'onclose' of undefined
```

This is the underlying cause of https://bugzilla.mozilla.org/show_bug.cgi?id=745829

**Comment by lloyd** [comment]
fix merged into train-2012.04.11.4 and dev.  now deployed on the latter, to be deployed on the former some time RSN.

**Comment by jrgm** [comment]
verified fixed for train-2012.04.11-8 on stage per https://bugzilla.mozilla.org/show_bug.cgi?id=745829#c2, and by invoking
navigator.id.get(function(){}) in the chrome/firefox browser.
